README.md

```diff
@@ -1,28 +1,72 @@
+This golden sample provides examples of the code structure, configuration, and best practices to use for developing your  Backbase journey.
 
-- `nrwl.angular-console`
-- `firsttris.vscode-jest-runner`.
 
-## Important
+## Table of Contents
+* [What is the app for](#what-is-the-app-for)
+* [Prerequisites](#prerequisites)
+* [Authentication details](#authentication-details)
+* [Generate an application](#generate-an-application)
+* [Generate a library](#generate-a-library)
+* [Development server](#development-server)
+* [Code scaffolding](#code-scaffolding)
+* [Build](#build)
+* [Tests](#tests)
+* [Understand your workspace](#understand-your-workspace)
+* [Running with docker](#running-with-docker)
+* [Package as a runnable Docker container](#package-as-a-runnable-docker-container)
+* [Further help](#further-help)
+
+
+
+
+
+## What is the app for
+- This project is a reference implementation to showcase best practices to use when building a new Angular SPA for Backbase components and libraries. The project is updated with the latest versions of the tools and libraries used. It shows guidelines about how to develop a web application for the frontend developers.
+- This prototype includes the following:
+    - Authentication
+    - Support of the multiple languages
+    - Integration with a theme
+    - Using existing journey installed from local NPM repository
+    - Configuring journeys
+    - Configuring entitlements
+    - Communication between journeys
+ 
+ ### What components are part of the app
+  - Auth module
+  - Locale selector for SPA
+  - Theme
+  - Entitlement
+  - Configuration of journeys
+  - Communication service
+  - Simplistic journey examples like transactions and transfer
+
+## Prerequisites
+For the best experience, the following VSCode extensions for Nx and Jest should be installed. 
+
+- [nrwl.angular-console](https://marketplace.visualstudio.com/items?itemName=nrwl.angular-console): to find and run all the possible Nx Commands.J
+- [firsttris.vscode-jest-runner](https://marketplace.visualstudio.com/items?itemName=firsttris.vscode-jest-runner): to easily run the isolated test as you are developing. 
+
+
+- On some AWS environments, due to specific WAF configuration, you may haved to use `http://0.0.0.0:4200/` when accessing the app locally, in order to successfully authenticate.
+
 
-On some AWS environments, due to specific WAF configuration, you might need to use `http://0.0.0.0:4200/` when accessing the app locally, in order to successfully authenticate.
 
 ## Authentication details
 
-### Prerequisites
+#### Prerequisites
 
 - Go through the [documentation page](https://community.backbase.com/documentation/foundation_angular/latest/authenticate_users) first to avoid any confusion with implementing authentication in the modelless app
```

Comment:
What about customers that have never known about our previous developments using model? Does it need any background?